activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceVariableControllerImpl.java

```diff
@@ -55,9 +63,23 @@ public ProcessInstanceVariableControllerImpl(ProcessInstanceVariableResourceAsse
     }
 
     @Override
-    public ResponseEntity<Void> setVariables(@PathVariable String processInstanceId,
-                                             @RequestBody SetProcessVariablesPayload setProcessVariablesPayload) {
+    public ResponseEntity<List<String>> setVariables(@PathVariable String processInstanceId,
+                                                     @RequestBody SetProcessVariablesPayload setProcessVariablesPayload) {
+        
+        ProcessInstance processInstance = processRuntime.processInstance(processInstanceId);
         setProcessVariablesPayload.setProcessInstanceId(processInstanceId);
+        
+        List<ActivitiException> activitiExceptions = processVariablesHelper
```

Comment:
this code block is duplicated in `ProcessInstanceVariableAdminControllerImpl`

Reply:
It's a general problem of controllers admin/not admin... Currently I do not know how to avoid duplication in a good way...